reconcile/utils/terraform_client.py

```diff
@@ -264,22 +266,22 @@ def log_plan_diff(
                 if action == "no-op":
                     logging.debug([action, name, resource_type, resource_name])
                     continue
-                # Ignore RDS modifications that are going to occur during the next
-                # maintenance window. This can be up to 7 days away and will cause
-                # unnecessary Terraform state updates until they complete.
-                if (
-                    action == "update"
-                    and resource_type == "aws_db_instance"
-                    and self._is_ignored_rds_modification(
-                        name, resource_name, resource_change
-                    )
-                ):
-                    logging.debug(
-                        f"Not setting should_apply for {resource_name} because the "
-                        f"only change is EngineVersion and that setting is in "
-                        f"PendingModifiedValues"
-                    )
-                    continue
+                if action == "update":
+                    if resource_type == "aws_db_instance":
+                        pending_rds_modifications = self._get_pending_rds_modification(
+                            name, resource_name, resource_change
+                        )
+                        # Ignore RDS modifications that are going to occur during the next
+                        # maintenance window. This can be up to 7 days away and will cause
+                        # unnecessary Terraform state updates until they complete.
+                        if pending_rds_modifications:
+                            logging.debug(
+                                f"Not setting should_apply for {resource_name} because"
```

Comment:
We're really skipping because the diff from Terraform only includes the values that have already been applied and are accounted for in `PendingModifiedValues`. Maybe we can make this log message a bit closer to that?

Reply:
@steveteahan, makes sense. I am going to update log messages a bit.

Reply:
Done. Thank you!